packages/strapi-plugin-content-manager/config/queries/bookshelf.js

```diff
@@ -1,64 +1,38 @@
 const _ = require('lodash');
+const { convertRestQueryParams, buildQuery } = require('strapi-utils');
 
 module.exports = {
-  find: async function (params, populate, raw = false) {
-    return this.query(function(qb) {
-      _.forEach(params.where, (where, key) => {
-        if (_.isArray(where.value) && where.symbol !== 'IN' && where.symbol !== 'NOT IN') {
-          for (const value in where.value) {
-            qb[parseInt(value) ? 'orWhere' : 'where'](key, where.symbol, where.value[value]);
-          }
-        } else {
-          qb.where(key, where.symbol, where.value);
-        }
-      });
+  find: async function(params, populate) {
+    const model = this;
 
-      if (params.sort) {
-        qb.orderBy(params.sort.key, params.sort.order);
-      }
-
-      if (params.skip) {
-        qb.offset(_.toNumber(params.skip));
-      }
+    const filters = convertRestQueryParams(params);
 
-      if (params.limit) {
-        qb.limit(_.toNumber(params.limit));
-      }
-    }).fetchAll({
-      withRelated: populate || this.associations.map(x => x.alias)
-    }).then(data => raw ? data.toJSON() : data);
+    return this.query(buildQuery({ model, filters }))
+      .fetchAll({
+        withRelated: populate || this.associations.map(x => x.alias),
+      })
+      .then(data => data.toJSON());
   },
 
-  count: async function (params = {}) {
-    return await this
-      .forge()
-      .query(qb => {
-        _.forEach(params.where, (where, key) => {
-          if (_.isArray(where.value)) {
-            for (const value in where.value) {
-              qb[value ? 'where' : 'orWhere'](key, where.symbol, where.value[value]);
-            }
-          } else {
-            qb.where(key, where.symbol, where.value);
-          }
-        });
-      })
-      .count();
+  count: async function(params = {}) {
+    const model = this;
+
+    const { where } = convertRestQueryParams(params);
+
+    return this.query(buildQuery({ model, filters: { where } })).count();
   },
 
-  search: async function (params, populate, raw = false) {
+  search: async function(params, populate, raw = false) {
```

Comment:
The `raw` parameter is still there... we should remove it completely or keep it, but we definitely need to harmonise 🙈